service/src/main/java/bio/terra/workspace/service/workspace/flight/RemoveUserFromWorkspaceFlight.java

```diff
@@ -24,8 +24,10 @@ public RemoveUserFromWorkspaceFlight(FlightMap inputParameters, Object applicati
         inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
     String userToRemove = inputParameters.get(WorkspaceFlightMapKeys.USER_TO_REMOVE, String.class);
     Optional<WsmIamRole> roleToRemove =
-        Optional.ofNullable(
-            inputParameters.get(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, WsmIamRole.class));
+        inputParameters.get(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, WsmIamRole.class) != null
```

Comment:
This doesn't make sense to me. I don't think there is a semantic difference from what it was before. Further, it is overcomplicated. I would suggest you skip all of the Optionals and simply do:
```
WsmIamRole roleToRemove = inputParameters.get(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, WsmIamRole.class);
```
and pass `roleRoleToRemove` into the `RemoveUserFromSamStep`. If it is null, you will pass null. If it is not, you will pass a WsmIamRole. If what is stored in the map is not a WsmIamRole, the retrieval from the map will throw.

Reply:
Sorry I wasn't clear. The commit you approved doesn't work. I pushed it to demonstrate the problem with storing enum -- as opposed to enum name -- in the flight.

Problem with storing enum is, Jackson says `Failed to deserialize value '"WRITER"' ` ([build scan](https://scans.gradle.com/s/q7osmkspgmaao/tests/:service:connectedTest/bio.terra.workspace.service.workspace.flight.RemoveUserFromWorkspaceFlightTest/removeUserFromWorkspaceFlightDoUndo()?top-execution=1)).

[This page](https://www.baeldung.com/jackson-serialize-enums#:~:text=By%20default%2C%20Jackson%20will%20use%20the%20Enum%20name%20to%20deserialize%20from%20JSON.) says:

> By default, Jackson will use the Enum name to deserialize from JSON.

So can I store enum name instead of enum? That's what we do for other enum flight inputs.

Reply:
Done, removed optional.

Discussed offline. Storing enum name appears to be simplest. It's unfortunate that we can't store enum as-is.